Quake/gl_vidsdl.c

```diff
@@ -417,10 +417,19 @@ static qboolean VID_SetMode (int width, int height, int refreshrate, int bpp, qb
 		if (vid_borderless.value)
 			flags |= SDL_WINDOW_BORDERLESS;
 		
-		draw_context = SDL_CreateWindow (caption, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, flags);
+		draw_context = SDL_CreateWindow (
```

Comment:
Why is this style difference necessary?

Reply:
a little bit force of habit I'm following since making Doom 3 (dhewm3), vkQuake3 and RBDOOM-3-BFG (that I'm currently working on fixing to run on macOS...) ... as in them, that is the style guide. 

And when I traced/read the code, It just happened to catch my eye and tweaked it :-)

Also, IMHO, usually it's easier to following this style guide in general, as sometimes, some names can become quite long ... so being able to read one per line, usually makes it fit in our window.

Reply:
Please revert

Reply:
> Please revert

done 👍🏻 (https://github.com/Novum/vkQuake/pull/266/commits/d4590bc4fd22514a1e633a688ccd0c855a857c79)